### PR TITLE
create header component, add custom font size and colors to config 

### PR DESCRIPTION
**Pull request by naura1835**

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+const Header = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const menuItems = [
+    { title: "Home", link: "/" },
+    { title: "About", link: "#" },
+    { title: "Products", link: "#" },
+  ];
+
+  console.log(isOpen);
```
Comment (ebinabo) [inline src/components/header/Header.js]: can we remove console logs before we merge PRs

File: src/components/header/Header.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { ReactComponent as Cart } from "../../assets/Cart.svg";
+
+import "./Header.scss";
```
Comment (ebinabo) [inline src/components/header/Header.js:6]: There's a vscode extension that renders scss as CSS, would it make sense to use that here so we just use CSS in the project?

File: src/index.css [hunk 3]
```diff
@@ -2,16 +2,21 @@
 @tailwind components;
 @tailwind utilities;
 
+html {
+  font-size: 100%;
+  box-sizing: border-box;
+}
+
+*,
```
Comment (ebinabo) [inline src/index.css]: no need to reset margins and padding 

`@tailwind base` does that for you already

**Review by ebinabo — CHANGES_REQUESTED** [review]

File: src/components/header/Header.js (hunk 2 above)
Comment (naura1835) [reply src/components/header/Header.js:6]: so basically writing scss but ut converts it to css? Also what is the name of the extension?

Comment (ebinabo) [reply src/components/header/Header.js:6]: Live Sass Compiler

**Comment by naura1835** [comment]
I've made the other changes and added new files. Please take a look  @ebinabo

**Review by ebinabo — APPROVED** [review]

**Pull request merged**